common/errors.go

```diff
@@ -1524,12 +1541,15 @@ type ErrEndpointClientSideException struct{ BaseError }
 
 const ErrCodeEndpointClientSideException = "ErrEndpointClientSideException"
 
-var NewErrEndpointClientSideException = func(cause error) error {
+func NewErrEndpointClientSideException(cause error) RetryableError {
 	return &ErrEndpointClientSideException{
-		BaseError{
+		BaseError: BaseError{
 			Code:    ErrCodeEndpointClientSideException,
-			Message: "client-side error when sending request to remote endpoint",
+			Message: "client-side error, etc.",
```

Comment:
This message indicates the error is initiated by the upstream, after request was sent to endpoint. The new message is too vague and wouldn't be useful for narrowing down.

In general not sure how to feel about error msgs, maybe we wouldn't really need them... could be a refactor for another day

Reply:
hmm makes sense. do you think error code is descriptive enough?

Reply:
In general I think so, I don't remember any two errors that had same code but diff message, which means even a reference code of Code -> Explanation would be sufficient vs bloating the memory + code with long strings (Imagine on a 10k RPS instance thousands of copies of such long strings are created in memory). The overall mb might be negligible.

Anyway we can think about removing Message concept in a separate refactor if it made sense